code/modules/virus2/disease2.dm

```diff
@@ -146,6 +146,8 @@ var/global/list/disease2_list = list()
 	disease2_list -= ID
 
 /datum/disease2/disease/proc/makerandom(var/list/str = list(), var/list/rob = list(), var/list/anti = list(), var/list/bad = list(), var/atom/source = null)
+	if(istype(src, /datum/disease2/disease/predefined))
```

Comment:
override `makerandom()` for `/datum/disease2/disease/predefined` instead

Reply:
oh duh